### DISCUSSION TITLE
2024-gucun-sakura/

### DISCUSSION THREAD

**Discussion by giscus** [n0vad3v/blog-comments#2] [Ideas]
# 2024-gucun-sakura/

最近樱花开了不少，于是决定去顾村公园走走，拍拍照，顺便用 Hondata 录制一下 Datalog，看看 AFM 曲线是否有必要做一些修正。 以下是一些简单的照片分享：

https://nova.moe/2024-gucun-sakura/

**Comment by Jsaeron** [comment]
😊风景很好~

**Comment by STRRL** [comment]
白丝点赞

**Comment by nl8590687** [comment]
是的呀，推特变成了X，没有以前的感觉了😂自己的博客永远是自己的博客 (你大爷也永远是你大爷，逃~)
